pyomo/core/tests/unit/test_check.py

```diff
@@ -89,7 +89,7 @@ def test_false(self):
         except ValueError:
             pass
 
-
+Simple = Scalar
```

Comment:
This is a test suite, I don't think we need the "backwards compatibility" alias.